libs/content/workspace-ffi/src/apple/ios/api.rs

```diff
@@ -16,12 +17,23 @@ use workspace_rs::tab::TabContent;
 use super::super::response::*;
 use super::response::*;
 use crate::apple::keyboard::UIKeys;
-use crate::WgpuWorkspace;
+use crate::{ExtendedInput, WgpuWorkspace};
 
 #[no_mangle]
 pub extern "C" fn ios_frame(obj: *mut c_void) -> IOSResponse {
     let obj = unsafe { &mut *(obj as *mut WgpuWorkspace) };
-    obj.frame().into()
+
+    let frame = obj.context.frame_nr();
+    let span = span!(target: "svg_editor", Level::TRACE, "request ios frame", frame);
```

Comment:
Just some thoughts: should this span target here be `svg_editor` if it is around something with more scope (the entire frame)?

Reply:
I unresolved this comment, the targets are determined from the name of the crate, feels weird to break that pattern and call it svg_editor here this point is definitely more general than the SVG editor

Reply:
okay i changed it so that the target is workspace and not canvas

Reply:
Right now we use the `workspace` span target name for stuff that exists in `workspace-ffi`. That feels a bit off, like there should be a separation between `workspace-ffi` and `workspace`.

Reply:
they'll now show up as `workspace-ffi` and `workspace` in the logs

Reply:
The filter just specifies anything that starts with workspace should end up in our logs

Reply:
Okay that sounds good then.